assignment2/fileresult.txt

```diff
@@ -0,0 +1 @@
+finder-app/writer: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=34eb3f99065a45418e5f6aadaf9a6f6688d43813, for GNU/Linux 3.2.0, not stripped
```

Comment:
@parthnpatel20 it looks like you didn't get the file cross compiled here, it still x86-64.